pkg/cloudprovider/gce/gce.go

```diff
@@ -251,18 +273,17 @@ func (gce *GCECloud) CreateTCPLoadBalancer(name, region string, externalIP net.I
 		Name:       name,
 		IPProtocol: "TCP",
 		PortRange:  fmt.Sprintf("%d-%d", minPort, maxPort),
-		Target:     pool,
```

Comment:
Yup, that is right. This was recently added with the multi-port work, so I'm not sure how much it was thought through, but PortRange is the only way to specify ports in a GCE forwarding rule.

Since each forwarding rule has its own IP and there's no way to specify multiple smaller ranges, I don't believe this is a bug. The iptables rules the proxy creates will only accept packets targeted to that IP that are actually one on the service's ports - others won't be handled by the iptables rules.